### DISCUSSION TITLE
Work prospects after completion

### DISCUSSION THREAD

**Discussion by MarkyGee84** [Cyfrin/foundry-full-course-cu#3043] [General]
Hey everyone, I just wanted to ask if I complete this course will it give me enough skill and knowledge to get out there and go straight to work and be proficient in solidity etc? Ive done some courses on another 'paid' platform and once complete I found myself not really able to start working for a client etc as it just didn't teach enough to do so if that makes sense. So yes my question is basically will this course give me everything I need without having to then move on to another course. TIA

**Comment by EngrPips** [comment]
Hello @MarkyGee84, I am sorry to break it to you, but no course will ever teach you everything you need to be successful because knowledge only comes from the willingness to f**k around and find out, but I can assure you this course will give you all the needed foundation to start f**king around and making yourself better and ready for the real world.
